homework-2/script.js

```diff
@@ -0,0 +1,60 @@
+// Output the numbers from 20 to 30 through the gap using a step of 0.5 (20 20.5 21 21.5….)
+
+function listIntegerFloatNumbers(x, y) {
+  let resultArray = [];
+  for (let i = x; i <= y; i += 0.5) {
+    resultArray.push(i);
+  }
+  return resultArray.join(" ");
+}
+console.log(listIntegerFloatNumbers(20, 30));
+
+// One dollar is worth 27 hryvnias. Output data with the calculation of the cost of 10, 20, 30... 100 dollars
+
+function listSumDollars(x, y) {
+  let resultArray = [];
+  let price = 27;
+  for (let i = x; i <= y; i += 10) {
+    resultArray.push(i * price);
+  }
+  return resultArray.toString();
+}
+console.log(listSumDollars(10, 100));
+
+// This is an integer. Output all integers from 1 to 100 whose square does not exceed N numbers
+
+function squaresList(x) {
+  let resultArray = [];
+  let squaresVariable = 0;
+  for (let i = 1; i <= x; i++) {
+    squaresVariable = i * i;
+    if (squaresVariable <= x) {
+      resultArray.push(squaresVariable);
+    }
+  }
+  return resultArray.toString();
+}
+console.log(squaresList(100));
+
+// This is an integer. Find out whether it is prime (prime is a number greater than 1 that has no divisors other than 1 and itself)
+
+function checkPrimeNumber(x) {
+  for (let i = 2; i < x; i++) {
+    if (x % i === 0) {
+      return false;
+    }
+  }
+  return x > 1;
+}
+console.log(checkPrimeNumber(13));
+
+// Given some number. Determine whether it is possible to obtain this number by reducing the number 3 to a certain power. (For example, the numbers 9, 81 can be obtained, and 13 cannot be obtained)
+
+function findNumberViaSquare(x) {
+  if (x % 3 === 0) {
+    return "The result of a power of 3";
+  } else {
+    return "Not result of a power of 3";
+  }
+}
```

Comment:
Not bad, but judging by the description, you should immediately understand that the function returns a boolean (true, false). And it should be called isSquareOfNumber(). And it must contain 2 parameters - the number itself and the square which you want to check
```javascript
isSquareOfNumber(27, 3) // true
isSquareOfNumber(6, 2) // false
```